cl/recap/tests.py

```diff
@@ -932,7 +926,7 @@ def test_simple_zip_upload(self, mock_get_name, mock_extract):
             self.assertEqual(new_pq.status, PROCESSING_STATUS.SUCCESSFUL)
 
         # Are the documents marked as available?
-        for doc in self.docs:
+        for doc in self.docs:  # is this correct.
```

Comment:
Sorry, was this a q for me or to yourself?

Reply:
it was for me.  sorry left over from testing